src/main/java/cloud/fogbow/ras/core/models/orders/AttachmentOrder.java

```diff
@@ -91,6 +91,7 @@ public String getVolumeOrderId() {
         return volumeOrderId;
     }
 
+    // TODO(chico) - Check it with the Team; Why does this implementation does not follow other(Compute, Volume, ...).
```

Comment:
The only thing that needs to be updated is the actual allocation. Allocation is only meaningful to some resource types, which need to provide an actual implementation for this method.

Reply:
Ok. done!